avakas/cli.py

```diff
@@ -112,7 +112,7 @@ def cli_set_version(prerelease=False,
     project = add_metadata(project, **kwargs)
     project.write()
 
-    print("Version set to %s" % version)
+    print("Version set to %s" % project.version)
```

Comment:
```suggestion
    print(f"Version set to {project.version}")
```
F-strings are for friends